Added the review for the Che 7 End-user guide once again. (#783)
codenvy/che-docs: src/main/pages/che-7/end-user-guide/proc_creating-a-workspace-from-code-sample.adoc

[NOTE]
====
For versions of link:https://github.com/minishift/minishift/releases[Minishift up to 1.34], apply the following change:

----
$ oc set env dc/che CHE_WORKSPACE_PLUGIN__REGISTRY__URL=https://che-plugin-registry.openshift.io/v3
----
====

. Log into your console at `++https://<IP-address>:8443/console/++`.

. Select the *mini-che* project:
+
* `++https://<IP-address>:8443/console/project/mini-che/overview++`

. Click through to the mini-che route link, which loads the *Dashboard* by default and prompts you to create a workspace.
+
* `++http://che-che.<IP-address>.nip.io/++`

==== OCP and OSD

> If using OCP or OSD, obtain the URL for the Che instance from your administrator.

Reviewer: ```suggestion
For use on OCP or OSD, get the URL for the Che instance from the cluster administrator.
```

+
This allows the user to make further configuration before starting the workspace.
+
image::workspaces/create-and-edit.png[Create and Edit]

=== Changing the workspace configuration

If you chose the second option, you can configure the workspace's YAML configuration, among other tasks:

. From the *Overview* tab, rename a workspace, if you forgot to do that when creating it. Or, toggle the ephemeral mode, or export your workspace configuration to a file or private cloud. The user can also delete a workspace from here.
+
image::workspaces/workspace-config-overview.png[Worksapce configuration overview]

// TODO https://github.com/eclipse/che/issues/13665 fix screenshot
. From the *Projects* tab, add more projects to an existing workspace:
+
image::workspaces/workspace-config-projects.png[Add projects to workspace]

. From the *Plugins* tab, enable or disable workspace plug-ins. For example, start with a generic Java-based stack, then later add support for Node or Python.

. From the *Editors* tab, enable or disable workspace editors. For example, to switch to the Che 6 editor, select the GWT IDE. The Che 7 editor is based on Theia.

[id="configure-devfile"]
[start=5]

> . From the *Devfile* tab, edit the workspace YAML configuration. You can also load a custom container from here, if you previously built one and pushed it to a container registry.  You can also add commands and sample projects, if you prefer entering code over clicking buttons.

Reviewer: ```suggestion
. From the *Devfile* tab, edit the workspace YAML configuration. Additionally, use th *Devfile* tab to:
+
** load a custom container that has been previously built and pushed to a container registry
** add commands and sample projects using source code
```

There are multiple ways to run a workspace:

> * From the btn:[Run]  button in the top-right corner of the Workspace configuration page, which will start the workspace but will NOT switch to it.

Reviewer: ```suggestion
* By clicking the btn:[Run] button in the top-right corner of the workspace configuration page, which starts the workspace but does not switch to it.
```

> * From the btn:[Open] button in the top-right corner of the Workspace configuration page, which will start the workspace and launch it in the browser.

Reviewer: ```suggestion
* By clicking the btn:[Open] button in the top-right corner of the workspace configuration page, which starts the workspace and launches it in the browser.
```
